frontend/components/webpack.common.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 module.exports = {
   entry: {
-    'vue-strap': './src/index.js',
+    'components': path.join(__dirname, 'src', 'index.js'),
```

Comment:
Is this what determines the `dist/` file name?

Also need to update in:
- .gitignore: `# Vue-strap coverage reports` → `# Vue components coverage reports`
- frontend/components/README.md: `<script src="path/to/vue-strap.js"></script>`
- frontend/components/package.json: `"main": "dist/vue-strap.js"`
- test/functional/testUtil/test.js: `NULL_WHITELIST = ignore().add(['vue-strap.min.js']);` — no test failed?

Reply:
> Is this what determines the dist/ file name?

nope, it's the entry file for webpack to bundle the stuff. Relative paths are dependent on the cwd, so just made it a little more robust here 

> Also need to update in:

missed quite a few `vue-strap` references, thanks!

Reply:
Then what determines the `dist/` file name?

Reply:
> no test failed?

nope since the file was renamed, it would have failed in the next release though 😓 😓 

I need to do a better ctrl+f for vuestrap 😓

Reply:
> Then what determines the `dist/` file name?

```
output: {
  filename: '[name].min.js',
},
```

in `webpack.build/dev.js`

Reply:
Where is `[name]` from?

Reply:
from the key value above for `components: path...`, so I guess it does determine it slightly 😁 

more info here https://webpack.js.org/configuration/output/

Reply:
Ok.